src/cswinrt/main.cpp

```diff
@@ -34,7 +34,7 @@ namespace cswinrt
         { "include", 0, option::no_max, "<prefix>", "One or more prefixes to include in projection" },
         { "exclude", 0, option::no_max, "<prefix>", "One or more prefixes to exclude from projection" },
         { "addition_exclude", 0, option::no_max, "<prefix>", "One or more namespace prefixes to exclude from the projection additions" },
-        { "target", 0, 1, "<net8.0|net7.0|net6.0|netstandard2.0>", "Target TFM for projection. Omit for compatibility with .NET 6." },
+        { "target", 0, 1, "<net8.0|net7.0|net6.0|netstandard2.0>", "Target TFM for projection (.NET 8 is the default)" },
```

Comment:
I was actually referring to this before.

```suggestion
        { "target", 0, 1, "<net8.0|netstandard2.0>", "Target TFM for projection (.NET 8 is the default)" },
```

Reply:
Oops I'm blind ahah